src/tests/nativeaot/SmokeTests/SharedLibraryDependencyLoading/SharedLibraryHost.cpp

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifdef TARGET_WINDOWS
+#include "windows.h"
+#else
+#include "dlfcn.h"
+#endif
+#include <cstdint>
+
+#ifndef TARGET_WINDOWS
+#define __stdcall
+#endif
+
+// typedef for shared lib exported methods
+using f_MultiplyIntegers = int32_t(__stdcall *)(int32_t, int32_t);
+
+#ifdef TARGET_WINDOWS
+int __cdecl main()
+#else
+int main(int argc, char* argv[])
+#endif
+{
+#ifdef TARGET_WINDOWS
+    // We need to include System32 to find system dependencies of SharedLibraryDependencyLoading.dll
```

Comment:
What system dependencies exist for this library? It is created in this repo and I don't see anything being linked in.

Reply:
SharedLibraryDependencyLoading.dll is the NativeAOT'd library. It has a dependency on bcrypt.